src/riak_cs_config.erl

```diff
@@ -103,6 +104,25 @@
 -include("s3_api.hrl").
 -include("list_objects.hrl").
 
+-define(NOTICE(Bool, Msg),
```

Comment:
`notice` is another log severity level than `warning` in `lager` (even it is rarely used).
It seems a little strange for me that `NOTICE` macro uses `warning` severity.